packages/docusaurus-mdx-loader/src/remark/transformLinks/index.js

```diff
@@ -46,22 +54,13 @@ function toAssetRequireNode({node, index, parent, filePath, requireAssetPath}) {
     : `./${relativeRequireAssetPath}`;
 
   const hrefProp = `require('${inlineMarkdownLinkFileLoader}${relativeRequireAssetPath}').default`;
+  const linkText = (node.children[0] && node.children[0].value) || '';
 
   node.type = 'jsx';
 
   node.value = `<a target="_blank" href={${hrefProp}} ${
     node.title ? `title={${node.title}}` : ''
-  } >`;
-
-  const linkText = (node.children[0] && node.children[0].value) || '';
-  delete node.children;
-
-  parent.children.splice(index + 1, 0, {
-    type: 'text',
-    value: linkText,
-  });
-
-  parent.children.splice(index + 2, 0, {type: 'jsx', value: '</a>'});
+  } >${linkText}</a>`;
```

Comment:
```suggestion
  }>${linkText}</a>`;
```

Wonder if we can sately remove this space here? Seems not useful to keep it and unusual?
This would modify the snapshots but should be fine